docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -51,79 +51,60 @@ def store_files(files):
     return files.to_sql(name="slack_files", con=create_db_connection())
 
 
-@graph
-def store_slack_files_in_sql():
+@graph_asset
+def slack_files_table():
     return store_files(fetch_files_from_slack())
-
-
-graph_asset = AssetsDefinition.from_graph(store_slack_files_in_sql)
 ```
 
 ### Defining basic dependencies for graph-backed assets
 
-The `from_graph` attribute on the `AssetsDefinition` object infers upstream and downstream asset dependencies from the graph definition provided. In the most simple case when the graph returns a singular output, Dagster infers the name of the graph to be the outputted asset key.
+Similar to with basic op-backed assets, Dagster infers the upstream assets from the names of the arguments to the decorated function.
 
-In the example below, Dagster creates an asset with key `middle_asset` from the `middle_asset` graph. Just like assets defined via <PyObject object="asset" decorator />, each argument to the decorated graph function is an upstream asset name. `middle_asset` depends on `upstream_asset`, and `downstream_asset` depends on `middle_asset`:
+The example below includes an asset named `middle_asset`. `middle_asset` depends on `upstream_asset`, and `downstream_asset` depends on `middle_asset`:
 
 ```python file=/concepts/assets/graph_backed_asset.py startafter=start_basic_dependencies endbefore=end_basic_dependencies
-from dagster import AssetsDefinition, asset, graph
+from dagster import asset, graph_asset, op
 
 
 @asset
 def upstream_asset():
     return 1
 
 
-@graph
-def middle_asset(upstream_asset):
-    return add_one(upstream_asset)
+@op
+def add_one(input_num):
+    return input_num + 1
 
 
-middle_asset = AssetsDefinition.from_graph(middle_asset)
+@op
+def multiply_by_two(input_num):
+    return input_num * 2
+
+
+@graph_asset
+def middle_asset(upstream_asset):
+    return multiply_by_two(add_one(upstream_asset))
 
 
 @asset
 def downstream_asset(middle_asset):
-    return middle_asset + 1
+    return middle_asset + 7
 ```
 
-When your graph returns multiple outputs, Dagster infers each output name to be the outputted asset key. In the below example, `two_assets_graph` accepts `upstream_asset` and outputs two assets, `first_asset` and `second_asset`:
+### Graph-backed multi-assets
```

Comment:
This section should include the specific decorator the user needs to do this. Could you add it pls?

Reply:
Sure thing